pkg/manager/member/pd_member_manager_test.go

```diff
@@ -183,6 +183,23 @@ func TestPDMemberManagerSyncCreate(t *testing.T) {
 			pdPeerSvcCreated: false,
 			setCreated:       false,
 		},
+		{
+			name: "patch pod container",
```

Comment:
Please cover the case where the fields in the additional container override the fields generated by TiDB Operator.

Reply:
Can we check the final result of the PD container?

Reply:
And also cover the case with a new additional container.

Reply:
`TestPDMemberManagerSyncCreate` can't check final result, i will add  code into `TestPDMemberManagerSyncUpdate`.